src/summary_api.py

```diff
@@ -0,0 +1,19 @@
+from fastapi import FastAPI
```

Comment:
Very minor nitpick: perhaps this project could be in a folder called `fastapi_project` like we're doing with Flask.